src/test/java/com/google/api/codegen/protoannotations/testdata/ruby_library_no_gapic_config.baseline

```diff
@@ -4981,6 +4967,25 @@ module Library
       ].freeze
 
 
+      RETURN_PATH_TEMPLATE = Google::Gax::PathTemplate.new(
+        "shelves/{shelf}/books/{book}/returns/{return}"
+      )
+
+      private_constant :RETURN_PATH_TEMPLATE
+
+      # Returns a fully-qualified return resource name string.
+      # @param shelf [String]
+      # @param book [String]
+      # @param return_ [String]
+      # @return [String]
+      def self.return_path shelf, book, return_
+        RETURN_PATH_TEMPLATE.render(
```

Comment:
This RETURN_PATH_TEMPLATE comes from the pre-existing resource entity defined in another_service.proto, and now it shows up in the MyProtoClient generated class!

Note that there were no changes to the proto test files, only the gapic-generator sources.